src/app/(data)/log/page.tsx

```diff
@@ -1,12 +1,14 @@
 import Sidebar from "@/components/common/Sidebar";
 import MultipleAxesCharts from "@/components/log/charts/MultipleAxesCharts";
-import SynchronisedCharts from "@/components/log/charts/SynchronisedCharts";
+import AttitudeCharts from "@/components/log/charts/AttitudeCharts";
+import StatusCharts from "@/components/log/charts/StatusCharts";
 
 export default function LogPage() {
   return (
     <div className="block">
       <MultipleAxesCharts />
-      <SynchronisedCharts numOfDatasets={4} />
+      <StatusCharts></StatusCharts>
+      <AttitudeCharts />
```

Comment:
```suggestion
      <StatusCharts />
      <AttitudeCharts />
```

통일성 등을 감안하면 같은 방식으로 표기해주어도 좋을 것 같습니다.